addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -1281,6 +1281,50 @@ def test_out_invoice_line_onchange_taxes_2_price_unit_tax_included(self):
             'amount_total': -2627.014,
         })
 
+    def test_payment_term_line_fiscal_position(self):
+        """Test the mapping of payment term line accounts with fiscal position."""
+        account_revenue_copy = self.company_data['default_account_revenue'].copy()
```

Comment:
Isn't the case for income/expense account already covered elsewhere?

Reply:
I can't find anything about it, let's keep it to be sure